ScoutSuite/core/cli_parser.py

```diff
@@ -239,9 +235,32 @@ def _init_common_args_parser(self):
                             help='Name of out-of-scope services.')
         parser.add_argument('--exceptions',
                             dest='exceptions',
-                            default=None,
-                            nargs='?',
+                            default=[None],
+                            nargs='+',
                             help='Exception file to use during analysis.')
+        parser.add_argument('--result-format',
+                            dest='result_format',
+                            default='json',
+                            type=str,
+                            choices=['json', 'sqlite'],
+                            help="[EXPERIMENTAL FEATURE] The database file format to use. JSON doesn't require a server to view the report, "
+                                 "but cannot be viewed if the result file is over 400mb.")
+        parser.add_argument('--serve',
```

Comment:
Can we put the server related stuff in a sub-parser or something like that?

Copied from #260, @zer0x64 :
> This is kinda of a touchy subject, because
>
> 1. Theorically, the subparsers are meant to be providers
I understand it's currently the case, but I don't think we ever said subparsers were meant to be only for providers :o 

> 2. The way it is done right now, because of the execution order, you need to log in before being able to serve the report. This would require some heavier changes for now than what it's actually worth, because it touches how to provider object is created

I guess we can create an issue if it implies too many changes, but the `--serve` argument doesn't really makes sense in my opinion, it's very confusing.

Reply:
Will open an issue.